tests/browser/display-names-prod.test.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  describe,
+  beforeAll,
+  afterAll,
+  it,
+  expect as globalExpected,
+} from 'vitest';
+import { readFileSync, readdirSync } from 'node:fs';
+import { join } from 'node:path';
+import { dirContentsToObject } from '@sku-private/test-utils';
+import {
+  scopeToFixture,
+  bundlers,
+  cleanup,
+  type BundlerValues,
+} from '@sku-private/testing-library';
+
+const { sku, fixturePath } = scopeToFixture('display-names-prod');
+
+describe('display-names-prod', () => {
+  describe.sequential.for(bundlers)('bundler: %s', (bundler) => {
+    const args: BundlerValues<string[]> = {
+      vite: ['--config', 'sku.config.vite.ts', '--experimental-bundler'],
+      webpack: [],
+    };
+
+    describe('build', () => {
+      beforeAll(async () => {
+        const build = await sku('build', args[bundler]);
+        globalExpected(
+          await build.findByText('Sku build complete'),
+        ).toBeInTheConsole();
+      });
+
+      it('should create build output', async ({ expect }) => {
+        const distDir = fixturePath('dist');
+        const files = await dirContentsToObject(distDir);
+        expect(files).toMatchSnapshot();
+      });
+
+      it('should add displayName assignments to built JavaScript', async ({
```

Comment:
Is there any advantage to checking the JavaScript, relative to snapshotting the HTML? I'd expect the HTML to show `undefined` dislaynames if the javascript was broken.

Additionally, we should run the same tests for `sku start` to ensure displaynames aren't there.